types/sumoselect/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for sumoselect 3.0
+// Project: https://github.com/HemantNegi/jquery.sumoselect#readme
+// Definitions by: nagamejun <https://github.com/nagamejun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery" />
+
+type SumoSelectEvents = 'sumo:opening' | 'sumo:opened' | 'sumo:closing' | 'sumo:closed';
+
+type Option = Partial<{
+    placeholder: string;
```

Comment:
Please, could you provide a default value for each option? Example

```suggestion
/**
 * @default 'Select Here'
 */
    placeholder: string;
```